--- Class.javajet	2010-08-11 16:00:49.218000000 +0200
+++ Class.javajet	2010-08-11 16:01:16.515625000 +0200
@@ -278,9 +278,9 @@
 
       <%}%>
     <%}%>
-  <%}%>
 <%@ include file="Class/declaredFieldGenFeature.insert.javajetinc" fail="silent" %>
 <%@ end %><%//Class/declaredFieldGenFeature.override.javajetinc%>
+  <%}%>
 <%}%>
 <%if (isImplementation && genClass.hasOffsetCorrection() && !genClass.getImplementedGenFeatures().isEmpty()) {%>
 	/**
